kobo/settings/base.py

```diff
@@ -454,6 +454,7 @@ def __init__(self, *args, **kwargs):
         }
 
 CELERY_BROKER_URL = os.environ.get('KPI_BROKER_URL', 'redis://localhost:6379/1')
+CELERY_RESULT_BACKEND = CELERY_BROKER_URL
```

Comment:
I see we do this already [in KoBoCAT](https://github.com/kobotoolbox/kobocat/blob/d91834ccb174da0fd0305f8e8ae11ed93cce10c0/onadata/settings/kc_environ.py#L51), but does it mean that we store all the results from every task forever? I just checked on HHI production, and there are 4400 keys in the Redis DB, containing stuff like:
```
127.0.0.1:6379[2]> GET celery-task-meta-62bd2cf9-1f53-4bf8-9e47-1a0c9554a1b3
"{\"status\": \"SUCCESS\", \"traceback\": null, \"result\": true, \"task_id\": \"62bd2cf9-1f53-4bf8-9e47-1a0c9554a1b3\", \"children\": []}"
```
I also saw in the [documentation](https://docs.celeryproject.org/en/v4.3.0/userguide/tasks.html?highlight=result#result-backends):
> Backends use resources to store and transmit results. To ensure that resources are released, you must eventually call get() or forget() on EVERY AsyncResult instance returned after calling a task.

Reply:
I thought I did :-(  You are totally right. The keys should be dropped / should expire.

Reply:
@jnm They are automatically removed after 1 day (by default) by `celery beat`. Obviously, we can lower this value.

https://docs.celeryproject.org/en/latest/userguide/configuration.html#result-expires
> A built-in periodic task will delete the results after this time (celery.backend_cleanup), assuming that celery beat is enabled. The task runs daily at 4am.

Reply:
Cool, I didn't know about that. Thanks. I think 1 day is fine